src/components/displayDocuments/renderQrCode.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "@emotion/styled";
+import QRCode from "qrcode.react";
+
+const QrContainer = styled.div`
```

Comment:
I dont think you need this contained if you just tell the arcode to take full space

Reply:
this qrContainer is for the button, not the actual qr code. i will rename it

Reply:
I dont understand qhy the container in this file is for a button in another file ? 🤔

Reply:
i saw wrongly, yes you are right